lib/easypost/shipment.rb

```diff
@@ -6,6 +6,12 @@ def get_rates(params={})
     return self
   end
 
+  def get_smartrates
+    response = EasyPost.make_request(:get, url + '/smartrate', @api_key)
+
+    return response
```

Comment:
The team is leaning toward having the client library provide idiomatic language access to our resources, so let's peel back the `results` layer of the onion on this endpoint, even if the wire API response is different.